iguana/3p_coins.json

```diff
@@ -39,7 +39,7 @@
     "git": {
       "repo": "https://github.com/sillyghost/AYAv2",
       "branch": "master",
-      "commit": "",
+      "commit": "aa6d65b",
```

Comment:
correct AYA commit is https://github.com/sillyghost/AYAv2/commit/fd94422aff2886919dc963d85c313df4dfb0d770

we should update it everywhere if it's different

Reply:
I just updated it based on readme and informed @ca333 about the difference. Will update this once he updates the readme